packages/admin/cms-admin/src/common/MasterMenu.tsx

```diff
@@ -4,8 +4,16 @@ import { RouteProps, useRouteMatch } from "react-router-dom";
 
 import { CurrentUserContext } from "../userPermissions/hooks/currentUser";
 
+// See UserPermissionsService.getAvailablePermissions in cms-api
+export enum Permission {
```

Comment:
Is Permission too generic? Maybe rename it to UserPermission in Admin and API?

Reply:
I'm not sure about this. First, it's not directly connected to the user at this place, a `UserPermission` is more what an actual user posesses. Second, if we have a `UserPermission` one might think that there exists another type of permission as well.

Furthermore the same logic would apply to `ContentScope`, so this should be called `UserContentScope` then. Does not sound good.

One practical issues: then we also have to rename the decorator to `RequiredUserPermission`, which makes the naming (also when talking about it) a whole lot bulkier.

And one technical issue: we also have an entity called `UserPermission`, this should be renamed as well (but to what?).

@nsams What's your take on this?

Reply:
Naming things is hard 😁

The reason I've brought this up is because we already have a `download` function that's part of the public API, which is very confusing. `Permission` could be a lot of things. Here, it is the permissions that are available for the core Comet modules. Maybe `AvailablePermissions` would be a better name for the enum? Or remove the enum at all, and use a type alias (yes, I know, I suggested the enum in the first place 😅)?

Reply:
I would not use an enum, I would even say simply hardcoding strings is Ok (like it was previously)

Reply:
e9fe697645484c079448f9a0016234ab4978a078